pkg/apiserver/proxy.go

```diff
@@ -151,6 +151,8 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	httpCode = http.StatusOK
 	newReq.Header = req.Header
+	newReq.ContentLength = req.ContentLength
+	newReq.TransferEncoding = req.TransferEncoding
```

Comment:
Looks like the reverse proxy strips the transfer encoding header.... isn't it a problem to send content length to the backend without transfer encoding?